packages/wonder-blocks-dropdown/components/single-select-menu.test.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import {mount} from "enzyme";
```

Comment:
Don't use `mount` directly. Avoid having to tidy up every mount by using our own `mount` and the handy `unmountAll` call. Otherwise, you run the risk of side effects between test cases because they share a DOM and there's no auto-unmounting between tests with enzyme directly.